### ISSUE TITLE
[Bug] Compile native, not support generic invoke

### ISSUE THREAD

**Issue by eye-gu** [apache/dubbo#14499]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dubbo/issues?q=is%3Aissue) and found no similar issues.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Dubbo Version

Dubbo version : 3.3.0-beta.1

### Steps to reproduce this issue

Compile native, not support generic invoke

### What you expected to happen

Compile native, support generic invoke

### Anything else

When the dubbo Maven plugin performs AOT preprocessing, there are no generic interfaces in the created proxy-config.json file

### Are you willing to submit a pull request to fix on your own?

- [X] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment]
There is no need to add GenericService to proxy-config.json

**Comment by eye-gu** [comment]
> There is no need to add GenericService to proxy-config.json

The exception is: com.oracle.svm.core.jdk.UnsupportedFeatureError: Proxy class defined by interfaces [interface com.alibaba.dubbo.rpc.service.GenericService, interface org.apache.dubbo.rpc.service.GenericService, interface org.apache.dubbo.rpc.service.EchoService, interface org.apache.dubbo.rpc.service.Destroyable] not found. Generating proxy classes at runtime is not supported. Proxy classes need to be defined at image build time by specifying the list of interfaces that they implement.

Add a `proxy-config.json` file, The content is:
```json
[
  {
    "interfaces": [
      "com.alibaba.dubbo.rpc.service.GenericService",
      "org.apache.dubbo.rpc.service.GenericService",
      "org.apache.dubbo.rpc.service.EchoService",
      "org.apache.dubbo.rpc.service.Destroyable"
    ]
  }
]
```

Successfully initiated generic invoke.

**Comment by AlbumenJ** [comment]
> > There is no need to add GenericService to proxy-config.json
> 
> The exception is: com.oracle.svm.core.jdk.UnsupportedFeatureError: Proxy class defined by interfaces [interface com.alibaba.dubbo.rpc.service.GenericService, interface org.apache.dubbo.rpc.service.GenericService, interface org.apache.dubbo.rpc.service.EchoService, interface org.apache.dubbo.rpc.service.Destroyable] not found. Generating proxy classes at runtime is not supported. Proxy classes need to be defined at image build time by specifying the list of interfaces that they implement.
> 
> Add a `proxy-config.json` file, The content is:
> 
> ```json
> [
>   {
>     "interfaces": [
>       "com.alibaba.dubbo.rpc.service.GenericService",
>       "org.apache.dubbo.rpc.service.GenericService",
>       "org.apache.dubbo.rpc.service.EchoService",
>       "org.apache.dubbo.rpc.service.Destroyable"
>     ]
>   }
> ]
> ```
> 
> Successfully initiated generic invoke.

@CrazyHZM PTAL

**Comment by eye-gu** [comment]
I'm afraid #14573  hasn't solved this problem. I tried fix it and you can check my PR #15139
